x-pack/plugins/snapshot_restore/public/application/components/policy_form/steps/step_logistics.tsx

```diff
@@ -21,13 +21,13 @@ import {
 } from '@elastic/eui';
 
 import { Repository } from '../../../../../common/types';
-import { CronEditor } from '../../../../shared_imports';
+import { CronEditor, SectionError } from '../../../../shared_imports';
 import { useServices } from '../../../app_context';
 import { DEFAULT_POLICY_SCHEDULE, DEFAULT_POLICY_FREQUENCY } from '../../../constants';
 import { useLoadRepositories } from '../../../services/http';
 import { linkToAddRepository } from '../../../services/navigation';
 import { documentationLinksService } from '../../../services/documentation';
-import { SectionLoading, SectionError } from '../../';
+import { SectionLoading } from '../../';
```

Comment:
I am not seeing in the diff that the SectionError file has been deleted, did I miss it?

Reply:
Here is the section_error diff

```diff
diff --git a/x-pack/plugins/snapshot_restore/public/application/components/section_error.tsx b/src/plugins/es_ui_shared/__packages_do_not_import__/authorization/components/section_error.tsx
similarity index 54%
rename from x-pack/plugins/snapshot_restore/public/application/components/section_error.tsx
rename to src/plugins/es_ui_shared/__packages_do_not_import__/authorization/components/section_error.tsx
index bd9e487967..3d56309ada 100644
--- a/x-pack/plugins/snapshot_restore/public/application/components/section_error.tsx
+++ b/src/plugins/es_ui_shared/__packages_do_not_import__/authorization/components/section_error.tsx
@@ -1,7 +1,20 @@
 /*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
  */

 import { EuiCallOut, EuiSpacer } from '@elastic/eui';
````